.github/workflows/frieeren-component-CD.yml

```
@@ -3,16 +3,31 @@ env:
+        options:
+          - patch # 0.0.1 -> 0.0.2
+          - minor # 0.0.1 -> 0.1.0
+          - major # 0.0.1 -> 1.0.0
   push:
     branches:
       - main
     paths:
       - packages/frieeren-components/**
 permissions:
   contents: write
+env:
```

actionlint (1.7.7)

key "env" is duplicated in workflow. previously defined at line:2,col:1

(syntax-check)